.travis.yml

```
@@ -35,20 +31,29 @@ before_install:
   - conda update --yes conda
 
 install:
+  # gjslint only runs in py27
+  - conda create --yes -n gjslint python=2.7 pip six
+  - source activate gjslint
+  - pip install https://github.com/google/closure-linter/archive/master.zip
```

You can collapse the line above and below this line into:

```suggestion
  - conda run -n gjslint pip install https://github.com/google/closure-linter/archive/master.zip
```